packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -292,46 +300,43 @@ export class AccountTrackerController extends StaticIntervalPollingControllerV1<
    *
    * @param networkClientId - Optional networkClientId to fetch a network client with
    */
-  refresh = async (networkClientId?: NetworkClientId) => {
+  async refresh(networkClientId?: NetworkClientId) {
```

Comment:
This feels more of a cleanup task to me rather than connected to converting to BaseController v2 — what are your thoughts about doing that in another PR?

Reply:
this is not a private function it's used in multiple places in metamask-mobile